btree/btree.h

```diff
@@ -270,6 +270,25 @@ void m0_btree_put(struct m0_btree *arbor, const struct m0_btree_rec *rec,
 		  const struct m0_btree_cb *cb, uint64_t flags,
 		  struct m0_btree_op *bop, struct m0_be_tx *tx);
 
+/**
+ * Updates the record in the tree. The callback is called with the location
+ * where the updated value should be written in the tree.
+ *
+ * @param arbor is the pointer to btree.
+ * @param rec   represents the record which needs to be updted. Note that,
+ *              user may or may not provide valid value but record should be
+ *              provided with valid key, key size and value size as this
+ *              information is needed for correct operation.
+ * @param cb    routine to be called to PUT the record.
+ * @param flags
```

Comment:
description for flags is missing.

Reply:
There is nothing to describe, I guess. Let me see if Radha thinks something should be added here.